### ISSUE TITLE
Bug-040: Screen reader does not announce blog tag drop dropdown options

### ISSUE THREAD

**Issue by StevenOyar** [StevenOyar/wk-6-StevenOyar-1#84]
**Related TC:** TC-099
**Related Requirement:** FR-074

---

**Severity:** Major
**Priority:** High
**Status:** Open

---

**Environment:**

* Screen Reader: **NVDA**  installed in my laptop
* OS: Windows 11 Pro
* Device: Hp Elitebook G6
* Resolution: 1920×1080
* URL: http://localhost:3000/

---

**Steps to Reproduce:**

1. Launch the application in Chrome.
2. Enable **NVDA** screen reader.
3. Navigate to the blog tag selection page.
4. Focus on the dropdown for choosing blog tags.

---

**Expected Result:**

Screen reader should announce:

* The dropdown label
* All available tag options as the user navigates

**Actual Result:**

NVDA does **not announce** the options in the blog tags dropdown.
User cannot hear what tags are present or selected.

---

**Evidence**

https://github.com/user-attachments/assets/1bcb9673-c182-4c11-b7fa-4edc3fce71c6

Reported by: @StevenOyar 
Date: 14/11/2025
